tests/test_validate_outputs.py

```diff
@@ -107,28 +107,28 @@ def test_calexp(self):
 
     def test_background(self):
         """Test background level."""
-        bkg = self.butler.get("calexpBackground", detector=self.detector, visit=self.visit)
+        bkg = self.butler.get("initial_pvi_background", detector=self.detector, visit=self.visit)
 
         bg0_arr = bkg.getImage().array
         bg_mean = bg0_arr.mean(dtype=np.float64)
         bg_std = bg0_arr.std(dtype=np.float64)
 
         for name, var, val in (
-                ("calexpBackground mean", bg_mean, 179.2836464883374),
-                ("calexpBackground stddev", bg_std, 0.8296105383233686),
+                ("initial_pvi_background mean", bg_mean, 392.74626436826),
+                ("initial_pvi_background stddev", bg_std, 1.8123839393979808),
         ):
             with self.subTest(name):
                 self.assertAlmostEqual(var, val, places=7, msg=name)
 
-    def test_ic_src(self):
-        """Test icSrc catalog."""
-        ic_src = self.butler.get("icSrc", detector=self.detector, visit=self.visit)
-        self.assertEqual(len(ic_src), 266)
+    def test_initial_psf_stars(self):
+        initial_psf_stars = self.butler.get("initial_psf_stars_detector",
+                                            detector=self.detector, visit=self.visit)
+        self.assertEqual(len(initial_psf_stars), 266)
 
-    def test_src(self):
-        """Test src catalog."""
-        src = self.butler.get("src", detector=self.detector, visit=self.visit)
-        self.assertEqual(len(src), 1363)
+    def test_initial_stars(self):
+        initial_stars = self.butler.get("initial_stars_detector",
+                                        detector=self.detector, visit=self.visit)
+        self.assertEqual(len(initial_stars), 446)
```

Comment:
Higher background and fewer stars is the right sign, I suppose, but again, quite a large change.